test/controllers/admin_controller_test.rb

```diff
@@ -2,8 +2,8 @@
 
 class AdminControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @user = users(:one)
-    post user_session_path, params: { 'user[email]' => @user.email, 'user[password]' => 'password' }
+    @edited_user = users(:regular_user)
```

Comment:
Nit: could we re-name `edited_user` to `user_to_modify`? Modify since we are both editing and deleting, and using the future tense since we will be modifying in the test.

Reply:
done